### PR TITLE
feat(front-admin): adding masterclass edition ✨

### PR DESCRIPTION
**Pull request by MisterAzix**
## Ticket
https://www.notion.so/ETQ-admin-JVP-modifier-une-masterclass-74c789d302804c8b90deaa8e2a23b47a

## Infos
image

### CODE REVIEW

File: apps/front-admin/src/features/masterclassEdit/hooks/useGetMasterclass.ts [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  id: Masterclass['id'],
+  access_token?: string
+): Promise<Masterclass> => {
+  const url = new URL(`/api/masterclass/${id}`, process.env.API_URL);
+
+  const response = await fetch(url, {
+    headers: {
+      Authorization: `Bearer ${access_token}`,
+    },
+  });
```
[the remark this answers is not in the document]
Comment (MisterAzix) [reply apps/front-admin/src/features/masterclassEdit/hooks/useGetMasterclass.ts:17]: This behavior is voluntary, and validation is done on the server side.

File: apps/front-admin/src/features/masterclassEdit/components/Molecules/DescriptionField.tsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
+import React, { useId } from 'react';
+import { IMasterclassForm } from '../../types';
+import { InputGroup } from '@hetic-saline-royale-academy/kit-ui';
+import { Controller } from 'react-hook-form';
+import { Control } from 'react-hook-form/dist/types';
+
+interface TitleFieldProps {
```
Comment (benoitfvr) [inline apps/front-admin/src/features/masterclassEdit/components/Molecules/DescriptionField.tsx]: DescriptionField ?

File: apps/front-admin/src/features/masterclassEdit/components/Molecules/DescriptionField.tsx [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+  control: Control<IMasterclassForm>;
+}
+
+const TitleField = ({ control }: TitleFieldProps) => {
```
Comment (benoitfvr) [inline apps/front-admin/src/features/masterclassEdit/components/Molecules/DescriptionField.tsx]: Same

File: apps/front-admin/src/features/masterclassEdit/components/Molecules/DescriptionField.tsx [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+          value={value}
+          multiline
+          rows={5}
+        />
+      )}
+    />
+  );
+};
+
+export default TitleField;
```
Comment (benoitfvr) [inline apps/front-admin/src/features/masterclassEdit/components/Molecules/DescriptionField.tsx]: Same

**Review by benoitfvr — DISMISSED** [review]

**Pull request merged**